packages/prettyhtml/test/api.js

```diff
@@ -0,0 +1,26 @@
+const test = require('ava')
+const prettyhtml = require('../')
+
```

Comment:
Could you add a test which only test the defaults? `prettyhtml test.html`

Reply:
Do you mean to test that, for example, prettyhtml('<div>Hello World</div>') returns '<div>Hello World</div>'?

Reply:
Yes, but with a more advanced example.